src/harness/reference_models/pre_iap_filtering/pre_iap_filtering.py

```diff
@@ -41,11 +43,15 @@ def preIapReferenceModel(protected_entities, sas_uut_fad, sas_test_harness_fads)
   inter_sas_duplicate_grant.interSasDuplicateGrantPurgeReferenceModel\
                                                (sas_uut_fad, sas_test_harness_fads)
 
-  # TODO
   # Invoke PPA, EXZ, GWPZ, and FSS+GWBL purge list reference models
+  list_of_fss_neighboring_gwbl = pre_iap_util.getFssNeighboringGwbl(protected_entities['gwblRecords']
```

Comment:
Why not just put this in the reference model below? I don't think the other parts of pre-IAP filtering need this result.

Reply:
This was not put inside the reference model because of the 'Note' in the WG4 document regarding the input to this model. The Note is
"Note: if an FSS station is given as input, it assumed that there is a GWBL station within 150 km and thus the FSS’ exclusion zone is activated. If there is no GWBL within 150 km, DO NOT include the FSS station in this function."

Please let us know if you would like this approach to be changed.

Reply:
Ahh, now I understand where this came from :) Now if only I could remember what I was thinking when I wrote that note... I don't see any good reason to follow the note now, but it's also not a problem to leave the code as-is.